daemon/daemon.go

```diff
@@ -123,6 +123,9 @@ func (d *Daemon) Init() error {
 	}
 
 	d.listener = listeners[0]
+	if l, ok := d.listener.(*net.UnixListener); ok {
+		d.listener = &ucrednixListener{l}
+	}
```

Comment:
yes; right now just for testing, for example,
```sh
go build -v -o srv ./cmd/snapd/ &&
 scp srv kvm.snappy: &&
 ssh -t kvm.snappy sudo /lib/systemd/systemd-activate -l 8080 ./srv```
is quite useful. Down the road doing this "properly" will need more complex auth support.